### ISSUE TITLE
Realm's EmbeddedObject?

### ISSUE THREAD

**Issue by aehlke** [mentrena/SyncKit#185]
aka "embedded objects". I take it that this requires special integration to support? It looks like it would require special handling and I don't see reference to it but was wondering if anyone had miraculously tested it already. Would be handy.

**Comment by mentrena** [comment]
Yes they wouldn't work out of the box. I've been reading on them as I wasn't familiar and it certainly looks possible to support them, it also looks like an interesting problem to solve.
